src/System.Device.Gpio/Interop/Unix/libgpiod/V2/Binding/Enums/GpiodEdgeEventType.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+
+namespace System.Device.Gpio.Interop.Unix.libgpiod.V2.Binding.Enums;
```

Comment:
NIT: Another comment for deep hirerarchy depth, I love your separation of types into folders logically, but that doesn't help much for intellisense and usability. I'd suggest to just keep everything on the same root V2 namespace we pick. Folder structure can remain to have things separate as you have here.